Kickstarter-iOS/AppDelegate.swift

```diff
@@ -219,6 +219,12 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
           .messageBannerViewController?.showBanner(with: success ? .success : .error, message: message)
       }
 
+    self.viewModel.outputs.configurePerimeterX
+      .observeValues {
+        AppEnvironment.current.apiService.perimeterXClient
```

Comment:
This is now where we call `start` on PerimeterX, not in `Service`. Thinking is more along the lines of their official instructions - https://docs.perimeterx.com/docs/ios-how-to-integrate-the-sdk-in-your-application-v3 (Step 5). Although the SDK instructions explicitly say to "You should only call this function once" and `appDidFinishLaunchingWithOptions` is called on every relaunch.
Some ambiguity there.

Reply:
The instructions also say "It's essential to call this function as early as possible in your application and before any URL request to your server" so this should be fine. I feel like they meant to say "only call this function once _per app launch_" 🤷🏻‍♂️

Reply:
Ah ok makes sense